src/search/components/JsonData/style.scss

```diff
@@ -0,0 +1,7 @@
+.JsonData .JsonData-json {
+  display: none;
+}
+
+.JsonData.JsonData-visible .JsonData-json {
```

Comment:
I'm not sure why I nested these, that's definitely why I added the namespace to the class. I like the 10/20 one most.

No namespace concerns me since we could get `.JsonData .PrettyJson pre` or something like that and we'd mess up our CSS.

It's a lot easier to use a namespace now and not have to worry about it than to discover that we've mucked it up later and try to retrace our steps.